assets/js/base/components/country-input/country-input.tsx

```diff
@@ -9,7 +9,7 @@ import classnames from 'classnames';
 /**
  * Internal dependencies
  */
-import { ValidatedSelect } from '../select';
```

Comment:
It looks like `ValidatedSelect` (and `Select`) weren't used anywhere else. Do you think we should delete them?

Reply:
That was a question I had :) Can we? I don't think these are exported so I think perhaps yes.

Reply:
If they are not used anywhere neither exported, I would delete them, yes. We can always recover them with git. :slightly_smiling_face: